index.js

```diff
@@ -721,6 +726,37 @@ class GithubScm extends Scm {
 
         return Promise.resolve(bellConfig);
     }
+
+    /**
+     * Resolve a pull request object based on the config
+     * @method getPrRef
+     * @param  {Object}   config            Configuration
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @return {Promise}
+     */
+    _getPrInfo(config) {
+        return this.lookupScmUri({
+            scmUri: config.scmUri,
+            token: config.token
+        }).then(scmInfo =>
+            this.breaker.runCommand({
+                action: 'get',
+                scopeType: 'pullRequests',
+                token: config.token,
+                params: {
+                    owner: scmInfo.owner,
+                    repo: scmInfo.repo,
+                    number: config.prNum
+                }
+            })
+        ).then(pullRequestInfo => ({
+            name: `PR-${pullRequestInfo.number}`,
+            ref: `pull/${pullRequestInfo.number}/merge`,
+            sha: pullRequestInfo.merge_commit_sha
```

Comment:
It should be `pullRequestInfo.head.sha`

I looked at this PR with 2 commits here: https://github.com/screwdriver-cd/meta-cli/pull/7/commits
And access through this: https://api.github.com/repos/screwdriver-cd/meta-cli/pulls/7

Looks like the latest commit sha is inside `pullRequestInfo.head.sha`. Not sure what the `merge_commit_sha` is. It might be the sha if the PR is merged.